[2024-03-06T10:58:58.418-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T10:58:58.425-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T10:58:58.425-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T10:58:58.439-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T10:58:58.443-0300] {standard_task_runner.py:60} INFO - Started process 18844 to run task
[2024-03-06T10:58:58.445-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmp583li5k_']
[2024-03-06T10:58:58.446-0300] {standard_task_runner.py:88} INFO - Job 12: Subtask TSLA
[2024-03-06T10:58:58.478-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T10:58:58.528-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T10:58:59.248-0300] {history.py:222} ERROR - TSLA: No price data found, symbol may be delisted (1h 2024-01-01 -> 2024-01-02)
[2024-03-06T10:58:59.250-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T10:58:59.255-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T135858, end_date=20240306T135859
[2024-03-06T10:58:59.303-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T10:58:59.314-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T12:04:08.592-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:04:08.703-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:04:08.704-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T12:04:08.770-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T12:04:08.815-0300] {standard_task_runner.py:60} INFO - Started process 39042 to run task
[2024-03-06T12:04:08.889-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmp93q0tv2m']
[2024-03-06T12:04:08.906-0300] {standard_task_runner.py:88} INFO - Job 3: Subtask TSLA
[2024-03-06T12:04:09.716-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T12:04:10.351-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T12:04:12.147-0300] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/dags/get_stoks.py", line 19, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/base.py", line 77, in history
    return self._lazy_load_price_history().history(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/scrapers/history.py", line 226, in history
    logger.debug(f'{self.ticker}: yfinance received OHLC data: {quotes.index[0]} -> {quotes.index[-1]}')
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 5389, in __getitem__
    return getitem(key)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/datetimelike.py", line 379, in __getitem__
    result = cast("Union[Self, DTScalarOrNaT]", super().__getitem__(key))
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/_mixins.py", line 284, in __getitem__
    result = self._ndarray[key]
IndexError: index 0 is out of bounds for axis 0 with size 0
[2024-03-06T12:04:12.213-0300] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T150408, end_date=20240306T150412
[2024-03-06T12:04:15.461-0300] {standard_task_runner.py:107} ERROR - Failed to execute job 3 for task TSLA (index 0 is out of bounds for axis 0 with size 0; 39042)
[2024-03-06T12:04:15.493-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-06T12:04:15.619-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T12:15:34.589-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:15:34.630-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:15:34.631-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T12:15:34.664-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T12:15:34.682-0300] {standard_task_runner.py:60} INFO - Started process 46535 to run task
[2024-03-06T12:15:34.697-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmp1239_x7w']
[2024-03-06T12:15:34.701-0300] {standard_task_runner.py:88} INFO - Job 144: Subtask TSLA
[2024-03-06T12:15:34.973-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T12:15:35.371-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T12:15:38.687-0300] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/dags/get_stoks.py", line 19, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/base.py", line 77, in history
    return self._lazy_load_price_history().history(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/scrapers/history.py", line 226, in history
    logger.debug(f'{self.ticker}: yfinance received OHLC data: {quotes.index[0]} -> {quotes.index[-1]}')
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 5389, in __getitem__
    return getitem(key)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/datetimelike.py", line 379, in __getitem__
    result = cast("Union[Self, DTScalarOrNaT]", super().__getitem__(key))
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/_mixins.py", line 284, in __getitem__
    result = self._ndarray[key]
IndexError: index 0 is out of bounds for axis 0 with size 0
[2024-03-06T12:15:40.975-0300] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T151534, end_date=20240306T151540
[2024-03-06T12:15:41.166-0300] {standard_task_runner.py:107} ERROR - Failed to execute job 144 for task TSLA (index 0 is out of bounds for axis 0 with size 0; 46535)
[2024-03-06T12:15:41.216-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-06T12:15:41.319-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T12:16:58.798-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:16:58.809-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T12:16:58.810-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T12:16:58.825-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T12:16:58.830-0300] {standard_task_runner.py:60} INFO - Started process 48606 to run task
[2024-03-06T12:16:58.832-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmpb1ai_b7b']
[2024-03-06T12:16:58.833-0300] {standard_task_runner.py:88} INFO - Job 203: Subtask TSLA
[2024-03-06T12:16:58.895-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T12:16:58.993-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T12:16:59.586-0300] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/dags/get_stoks.py", line 19, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/base.py", line 77, in history
    return self._lazy_load_price_history().history(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/utils.py", line 103, in wrapper
    result = func(*args, **kwargs)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/yfinance/scrapers/history.py", line 226, in history
    logger.debug(f'{self.ticker}: yfinance received OHLC data: {quotes.index[0]} -> {quotes.index[-1]}')
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 5389, in __getitem__
    return getitem(key)
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/datetimelike.py", line 379, in __getitem__
    result = cast("Union[Self, DTScalarOrNaT]", super().__getitem__(key))
  File "/home/luan/Documents/airflow-executores-local-celery/venv/lib/python3.9/site-packages/pandas/core/arrays/_mixins.py", line 284, in __getitem__
    result = self._ndarray[key]
IndexError: index 0 is out of bounds for axis 0 with size 0
[2024-03-06T12:16:59.601-0300] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T151658, end_date=20240306T151659
[2024-03-06T12:16:59.616-0300] {standard_task_runner.py:107} ERROR - Failed to execute job 203 for task TSLA (index 0 is out of bounds for axis 0 with size 0; 48606)
[2024-03-06T12:16:59.649-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-06T12:16:59.671-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T14:38:34.310-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:38:34.323-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:38:34.324-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T14:38:34.338-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T14:38:34.342-0300] {standard_task_runner.py:60} INFO - Started process 82606 to run task
[2024-03-06T14:38:34.344-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmpejyj227j']
[2024-03-06T14:38:34.346-0300] {standard_task_runner.py:88} INFO - Job 254: Subtask TSLA
[2024-03-06T14:38:34.412-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T14:38:34.507-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T14:38:36.873-0300] {history.py:222} ERROR - TSLA: No price data found, symbol may be delisted (1h 2024-01-01 -> 2024-01-02)
[2024-03-06T14:38:36.875-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T14:38:36.890-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T173834, end_date=20240306T173836
[2024-03-06T14:38:36.933-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T14:38:36.957-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T14:49:00.720-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:49:00.732-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:49:00.733-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T14:49:00.748-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T14:49:00.752-0300] {standard_task_runner.py:60} INFO - Started process 86498 to run task
[2024-03-06T14:49:00.754-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmpd31f355x']
[2024-03-06T14:49:00.756-0300] {standard_task_runner.py:88} INFO - Job 283: Subtask TSLA
[2024-03-06T14:49:00.818-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T14:49:00.916-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T14:49:01.393-0300] {history.py:222} ERROR - TSLA: No price data found, symbol may be delisted (1h 2024-01-01 -> 2024-01-02)
[2024-03-06T14:49:01.397-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T14:49:01.412-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T174900, end_date=20240306T174901
[2024-03-06T14:49:01.450-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T14:49:01.471-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T14:59:50.524-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:59:50.537-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T14:59:50.537-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T14:59:50.553-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T14:59:50.557-0300] {standard_task_runner.py:60} INFO - Started process 90659 to run task
[2024-03-06T14:59:50.562-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmpu4xvxtl7']
[2024-03-06T14:59:50.564-0300] {standard_task_runner.py:88} INFO - Job 312: Subtask TSLA
[2024-03-06T14:59:50.634-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T14:59:50.733-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T14:59:52.053-0300] {history.py:222} ERROR - TSLA: No price data found, symbol may be delisted (1h 2024-01-01 -> 2024-01-02)
[2024-03-06T15:00:02.062-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T15:00:02.084-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T175950, end_date=20240306T180002
[2024-03-06T15:00:02.141-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T15:00:02.165-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T15:48:16.917-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T15:48:16.928-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-06T15:48:16.929-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T15:48:16.946-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): TSLA> on 2024-01-02 00:00:00+00:00
[2024-03-06T15:48:16.952-0300] {standard_task_runner.py:60} INFO - Started process 104263 to run task
[2024-03-06T15:48:16.955-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'TSLA', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '360', '--raw', '--subdir', 'DAGS_FOLDER/get_stoks.py', '--cfg-path', '/tmp/tmpjf9jml55']
[2024-03-06T15:48:16.956-0300] {standard_task_runner.py:88} INFO - Job 360: Subtask TSLA
[2024-03-06T15:48:17.031-0300] {task_command.py:423} INFO - Running <TaskInstance: get_stocks_dag.TSLA scheduled__2024-01-02T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T15:48:17.132-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_stocks_dag' AIRFLOW_CTX_TASK_ID='TSLA' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-06T15:48:18.255-0300] {history.py:222} ERROR - TSLA: No price data found, symbol may be delisted (1h 2024-01-01 -> 2024-01-02)
[2024-03-06T15:48:28.267-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T15:48:28.284-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=TSLA, execution_date=20240102T000000, start_date=20240306T184816, end_date=20240306T184828
[2024-03-06T15:48:28.333-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T15:48:28.354-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
